apps/meteor/client/views/omnichannel/analytics/InterchangeableChart.tsx

```diff
@@ -51,9 +66,16 @@ const InterchangeableChart = ({ departmentId, dateRange, chartName, ...props })
 			if (!result?.chartLabel || !result?.dataLabels || !result?.dataPoints) {
 				throw new Error('Error! fetching chart data. Details: livechat:getAnalyticsChartData => Missing Data');
 			}
-			context.current = await drawLineChart(canvas.current, context.current, [result.chartLabel], result.dataLabels, [result.dataPoints], {
-				tooltipCallbacks,
-			});
+			context.current = await drawLineChart(
+				canvas.current as HTMLCanvasElement,
```

Comment:
You can check if canvas.current is valid before doing this, so you don't typecast a null to this element